UPGRADE-14.0.md

```diff
@@ -113,6 +113,55 @@ Follow the instructions in relevant sections, e.g. `shopsys/coding-standards` or
     -   see #project-base-diff to update your project
 -   replace custom application bootstrapping with symfony/runtime ([#2914](https://github.com/shopsys/shopsys/pull/2914))
     -   see #project-base-diff to update your project
+-   enable crons to be run at specified times the same as crons ()
```

Comment:
what am I missing from the upgrade notes:
- `CronTimeInterface::getTimeMinutes` has now return typehint
- `CronTimeInterface::getTimeHours` has now return typehint
- `DateTimeHelper::createFromFormat` has now strict typehints
- `FeedController` constructor has changed
- `FeedRegistry::$feedsByType` was removed, there is a new `$feedConfigsByName` property instead
- `CronConfig::getAllCronModuleConfigs` has now return typehint
- `CronConfig::getCronModuleConfigsByTime` has now return typehint
- `CronConfig::getCronModuleConfigByServiceId` has now strict typehints
- `CronTimeResolver::isValidAtTime`  has now return typehint
- `CronTimeResolver::isMatchWithTimeString`  has now strict typehints
- `CronTimeResolver::validateTimeString`  has now strict typehints